tests/test_service.py

```diff
@@ -440,5 +440,61 @@ def test_load_api_with_awp_root_2_no_gatebin():
     assert dpf_inner_path in serv._grpc_client_path
 
 
+@pytest.fixture(autouse=False, scope="function")
+def clean_up(request):
```

Comment:
@cbellot000 Could we call this something more explicit such as "set context back to premium" or "entry_context"?